### ISSUE TITLE
Add support for package registry interaction

### ISSUE THREAD

**Issue by lionelvillard** [apache/incubator-openwhisk-wskdeploy#137]
`wskdeploy` should ease the interaction with public and private OpenWhisk package registries. This proposal is about adding the following commands:

- publish: publishes packages and notify registry
- install: installs the dependencies describes in the manifest. No registry interaction.
- unpublish: unpublishes packages

A separate effort is underway to provide an experimental public registry (more information about it soon). 

The issue depends on #41.  

- [ ] Publish command #154

**Comment by pritidesai** [comment]
@lionelvillard is this still relevant?

**Comment by lionelvillard** [comment]
Nope. closing.
